modular_citadel/code/modules/mob/living/carbon/human/species_types/ipc.dm

```diff
@@ -0,0 +1,44 @@
+/datum/species/ipc
+	name = "IPC"
+	id = "ipc"
+	say_mod = "beeps"
+	default_color = "00FF00"
+	blacklisted = 0
+	sexes = 0
+	species_traits = list(MUTCOLORS,NOEYES)
+	mutant_bodyparts = list("ipc_screen")
+	default_features = list("ipc_screen" = "Blank")
+	species_traits = list(MUTCOLORS)
```

Comment:
Duplicate of above species_traits. this results in white eyes rendering over IPC heads. Please remove